index.html

```diff
@@ -0,0 +1,163 @@
+    <link rel="stylesheet" href="./styles/index.css">
+</head>
+<body>
+    <header class="header row">
+        <nav class="header__nav container">
+            <div class="header__nav_logo cell">
+                <img src="./images/logo.svg" alt="logo">
+            </div>
+            <ul class="header__nav_list cell">
+                <li class="header__nav_list-item">The Team</li>
+                <li class="header__nav_list-item">Pricing</li>
+                <li class="header__nav_list-item">Features</li>
+            </ul>
+        </nav>
+        <section class="header__section container">
+            <div class="header__section_text cell">
+                <h1 class="header__section_text-title">Beautiful Free Nova template</h1>
+                <h4 class="header__section_text-description">A top notch premium quality template for your next web project.</h4>
+                <button class="header__section_text-button">Download FREE!</button>
+            </div>
+            <div class="header__section_image cell">
+                <img src="./images/screen.png" alt="screen" class="screen__image">
+        </section>
+    </header>
+    <main class="features row" id="Features">
+        <section class="features__section container">
+            <h2 class="features__section_title ">Features</h2>
+            <div class="features__section_box ">
+                <div class="features__section_box-feature">
+                    <div class="feature-icon">
+                        <img src="./images/responsive.svg" alt="responsive" >
+                    </div>
+                    <h5 class="feature-text">FULLY RESPONSIVE</h5>
+                    <p class="feature-subtext">Looks amazing on any device: smartphone, tablet, laptop and desktop.</p>
+                </div>
+                <div class="features__section_box-feature">
+                    <div class="feature-icon">
+                        <img src="./images/customizable.svg" alt="customizable" >
+                    </div>
+                    <h5 class="feature-text">CUSTOMIZABLE</h5>
+                    <p class="feature-subtext">Change the colors, pictures or any of the sections to suit your needs.</p>
+                </div>
+                <div class="features__section_box-feature">
+                    <div class="feature-icon">
+                        <img src="./images/design.svg" alt="design" >
+                    </div>
+                    <h5 class="feature-text">SLICK AND BEAUTIFUL DESIGN</h5>
+                    <p class="feature-subtext">Trendy and fresh design, fits any website.</p>
+                </div>
+            </div>
+        </section>
+    </main>
+    <section class="logos row">
+        <div class="logos__container container">
+            <img src="./images/logos.png" alt="logos" class="logos__image">
+        </div>
+    </section>
+    <section class="design row">
+        <aside class="desing__container">
```

Comment:
Tutaj bym wykorzystał  `<figure>`, bo to część artykułu :)

PS. Nie jest to błąd - raczej moja sugestia jak ja bym to zrobił :)